tests/README.md

```diff
@@ -1,6 +1,35 @@
 # `stackstorm-ha` Helm chart Tests
 
-This directory contains Helm chart tests, powered by [BATS](https://github.com/sstephenson/bats) Bash Automated Testing System.
+This directory contains Helm chart integration and unit tests (under `tests/integration/` and `tests/unit/` respectively).
+
+## Unit tests
+
+Unit tests (under `tests/unit/`) use [`helm-unittest`](https://github.com/quintush/helm-unittest).
+`helm-unittest` uses a yaml-based test file to ensure that the templates generate expected features.
+For example, they can ensure that custom annotations are applied consistently to all of the deployments.
+Unit tests do not require a running kubernetes cluster.
+
+Before running unit tests, install the `helm-unittest` plugin and ensure you have sub-charts installed:
+```
+helm plugin install https://github.com/quintush/helm-unittest
+helm dependency update
+```
+
+To run the tests manually from the chart's root dir:
+```
+helm unittest --helm3 tests/unit/*_test.yaml .
+```
+
+Note that `helm-unittest` still defaults to helm 2, so you must pass `--helm3` or `-3` for short.
+You must also tell it where the unit tests are with `-f` because we keep unit tests in the `tests/unit` directory.
```

Comment:
OK. I moved the tests to `tests/unit`.
That means we have to tell `helm-unittest` where to find the test files (by default it looks for `tests/*_test.yaml`).
Sadly `helm-unittest` has no config file so every invocation will have to specify which files to test with.

Reply:
@armab Can we just keep the unit tests directly under `tests/`? If we don't I'm unlikely to run them locally because specifying the path every time would be annoying. If we leave them in `tests/unit` I'll just let CI run the tests for me.

Reply:
I moved them back to `tests/` and documented the file name scheme: `tests/unit-name_your_test.yaml`.
This will keep the unit tests sorted within the `tests/` directory without making it awkward to run the unit tests.

Reply:
I think another CLI param is a reasonable price to pay to structure them in a usual way. Replied below.

Reply:
done